src/BetterKV.ts

```diff
@@ -11,8 +11,8 @@ export declare interface BetterKVGetOptions {
 }
 
 export declare interface BetterKVWithMetadata<V, M> {
-  value: V;
-  metadata: M;
+  value: V | null;
```

Comment:
These two I'm not sure about - the original KV type has them as `V | null`, and in my head nothing is preventing me from calling `.getWithMetadata` where the value is `null`, or the metadata is `null`, but not both (so it's not a simple `null` response.

This feels like a somewhat breaking change, from a typecheck perspective.